src/utils/nonNull.ts

```diff
@@ -36,8 +36,8 @@ export function nonNullOrEmptyValue(
 ): string {
   if (!value) {
     throw new Error(
-      l10n.t('Internal error: Expected value to be neither null, undefined, nor empty') +
-        (propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''),
+      `${errorMessage.internalErrorNullOrUnderinedOrEmpty} +
+      ${propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''}`,
```

Comment:
no need to add translations for line `errorMessage.internalErrorNullOrUnderinedOrEmpty` to `message.ts` because it will be removed just revert it as it was

Reply:
removed